### PR TITLE
Implement command aliases

### PR DESCRIPTION
**Pull request by ks129**
- Add alias parsing to `Context.findCommand`.
- Add new function to `Subcommand`: `AddAliases` that add new alias(es) to command.
- Added `Aliases` property to `MethodContext`

I don't know how to get this fork and branch as a package, so I'm unable to test it. Can anyone test. Logically should everything working probably.

### CODE REVIEW

**Comment by ks129** [comment]
Got working my fork. I can now say that aliases work.

File: bot/subcommand.go [hunk 1]
```diff
@@ -390,6 +390,30 @@ func (sub *Subcommand) SetPlumb(methodName string) {
...
+// for safety reasons (duplicates).
+func (sub *Subcommand) AddAliases(commandName string, aliases ...string) {
+	// Get command
+	command := sub.FindCommand(commandName)
+
+	// Merge current aliases for duplicate removing
+	aliases = append(aliases, command.Aliases...)
+
+	// Remove all duplicate aliases
+	keys := make(map[string]bool)
```
Comment (diamondburned) [inline bot/subcommand.go:404]: This seems kind of excessive. I think it would be fine to leave it to the developer.

**Comment by ks129** [comment]
@diamondburned Removed check for duplicates.

File: bot/subcommand.go [hunk 2]
```diff
@@ -390,6 +390,15 @@ func (sub *Subcommand) SetPlumb(methodName string) {
 	sub.plumbed = sub.FindCommand(methodName)
 }
 
+// AddAliases add alias(es) to specific command (defined with commandName).
+func (sub *Subcommand) AddAliases(commandName string, aliases ...string) {
+	// Get command
+	command := sub.FindCommand(commandName)
+
+	// Write new listing of aliases
+	command.Aliases = append(command.Aliases, aliases)
```
Comment (diamondburned) [inline bot/subcommand.go:399]: Does this compile? Shouldn't this be `aliases...`?

**Comment by ks129** [comment]
@diamondburned ... Looks like my GoLand didn't give me this error. Interesting. However, now this is fixed.

**Pull request merged**